geomstats/backend/pytorch/random.py

```diff
@@ -7,6 +7,12 @@ def rand(*args, **kwargs):
     return torch.rand(*args, **kwargs)
 
 
+def randn(*args, **kwargs):
+    """Sample over a normal N(0,1)
```

Comment:
No need for docstrings in the backends, but thank you! :)

Reply:
Although, if you do want one here, then it should be on one line, and end with a ".".